pytorch_lightning/overrides/data_parallel.py

```diff
@@ -24,7 +24,7 @@
 from torch.nn.parallel._functions import Gather
 
 from pytorch_lightning.core.step_result import Result
-from pytorch_lightning.utilities.warning_utils import WarningCache
+from pytorch_lightning.utilities.warnings import WarningCache
```

Comment:
should the name for the module be different from stdlib modules? https://github.com/PyTorchLightning/pytorch-lightning/issues/4436

Reply:
well, do you have a suggestion for a better name? :]

Reply:
pl_warnings and pl_argparse maybe :sweat_smile:

Reply:
in such case, we would need to. add pl to everything :D